rosbag2_py/test/test_convert.py

```diff
@@ -64,22 +65,23 @@ def test_output_bags_not_a_list(self):
     def test_basic_convert(self):
         # This test is just to test that the rosbag2_py wrapper parses input
         # It is not a comprehensive test of bag_rewrite.
+        storage_id = get_default_storage_id()
         bag_a_path = RESOURCES_PATH / 'convert_a'
         bag_b_path = RESOURCES_PATH / 'convert_b'
         output_uri_1 = self.tmp_path / 'converted_1'
         output_uri_2 = self.tmp_path / 'converted_2'
         input_options = [
             StorageOptions(uri=str(bag_a_path)),
-            StorageOptions(uri=str(bag_b_path), storage_id='sqlite3'),
+            StorageOptions(uri=str(bag_b_path)),
```

Comment:
Shouldn't we still pass `storage_id` as initialization parameter for `StorageOption`?

Reply:
Since we added the ability to autodetect storage for inputs, it's not necessary to specify. I was thinking that at some point we might convert test bags to mcap, so using autodetect as much as possible would help with that if we ever do it.